### PR TITLE
gh-119180: annotationlib: Fix __all__, other minor fixes

### PR DESCRIPTION
**Pull request by JelleZijlstra**
* Issue: gh-119180

### CODE REVIEW

File: Lib/annotationlib.py [hunk 1]
```diff
@@ -6,7 +6,14 @@
 import sys
 import types
 
-__all__ = ["Format", "ForwardRef", "call_annotate_function", "get_annotations"]
+__all__ = [
+    "Format",
+    "ForwardRef",
+    "call_annotate_function",
+    "call_evaluate_function",
+    "get_annotate_function",
```
Comment (JelleZijlstra) [inline Lib/annotationlib.py:14]: These two were missing. The rest of the changes in this file are just formatting.

File: Lib/test/test_annotationlib.py [hunk 2]
```diff
@@ -867,6 +881,13 @@ class D(metaclass=Meta):
...
                         if c.expected_annotations:
-                            self.assertEqual(annotate_func(Format.VALUE), c.expected_annotations)
+                            self.assertEqual(
+                                annotate_func(Format.VALUE), c.expected_annotations
+                            )
                         else:
                             self.assertIs(annotate_func, None)
+
+
+class TestAnnotationLib(unittest.TestCase):
```
Comment (JelleZijlstra) [inline Lib/test/test_annotationlib.py:891]: This is the only non-formatting change in this file.

**Review by AA-Turner — COMMENTED** [review]
This is the first stdlib module we're proposing to use autoformatting for, is it worth checking on Discourse? Autoformatters are much more opionated than PEP 8, which may be a surprising experience for contributors (note @encukou ran into an issue with a similar tool just whilst making a change to a CPython-only docs extension).

A

**Comment by JelleZijlstra** [comment]
I hoped it wouldn't be controversial since we're already doing it for some of the `Tools/` code. I'm not interested in discussing this issue on Discourse; if there's disagreement, I'll just take out the autoformatting.

**Pull request merged**